ReactWindows/ReactNative.Net46/Views/Image/ReactImageManager.cs

```diff
@@ -295,10 +296,8 @@ private void SetUriFromSingleSource(Border view, string source)
                     status => OnImageStatusUpdate(view, status),
                     _ => OnImageFailed(view));
 
-                using (var stream = BitmapImageHelpers.GetStreamAsync(source))
-                {
-                    image.StreamSource = stream;
-                }
+                var stream = BitmapImageHelpers.GetStreamAsync(source);
+                image.StreamSource = stream;
```

Comment:
Do you still need to dispose the stream after EndInit?

Reply:
According to the docs for BitmapImage.streamSource: 

> Set the CacheOption property to BitmapCacheOption.OnLoad if you wish to close the stream after the BitmapImage is created. The default OnDemand cache option retains access to the stream until the bitmap is needed, and cleanup is handled by the garbage collector.

https://msdn.microsoft.com/en-us/library/system.windows.media.imaging.bitmapimage.streamsource%28v=vs.110%29.aspx